packages/owl-bot/cron.yaml

```diff
@@ -0,0 +1,7 @@
+cron:
+  - name: owl-bot-sync-label
+    schedule: 10 3 * * *
+    description: owl-bot syncing labels
+    params:
+      cron_type: "repository"
+      syncLabels: true
```

Comment:
Why isn't there an existing entry for `scanGithubForConfigs()`?

Reply:
My guess is `cron.yaml` is relatively a new thing. The scheduler job `owl-bot-scan-googleapis` might have created before `cron.yaml` is a thing.

I can create the entry if you like.

Reply:
Added an entry